### PR TITLE
Add JD40 keymap for myoung34

### PR DESCRIPTION
**Pull request by myoung34**
## Description

Add JD40 keymap for my layout

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/jd40/keymaps/myoung34/keymap.c [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  } else {
+      //gh60_wasd_leds_off();
+  }
+
+  if (layer & (1 << 2)) {
+      //gh60_esc_led_on();
+  } else {
+      //gh60_esc_led_off();
+  }
+};
```
Comment (fauxpark) [inline keyboards/jd40/keymaps/myoung34/keymap.c:45]:
```suggestion
```

**Comment by myoung34** [comment]
@fauxpark I saw your other PR about `info.json`, is that whats blocking here?
Im just trying to add a keymap, it looks like `helix` is missing it in this action failure

`keyboards/jd40/info.json` does exist though

**Comment by myoung34** [comment]
@fauxpark I saw your other PR about `info.json` [here](https://github.com/qmk/qmk_firmware/pull/9549), is that whats blocking here?
Im just trying to add a keymap, it looks like `helix` is missing it in this action failure

`keyboards/jd40/info.json` does exist though

**Comment by fauxpark** [comment]
The PR lint error is because the difference between your branch and current master includes #9528, which fixes some (but not all) of the info.json errors on the Helix boards. If you rebased your branch onto current master, the difference would only include your commits, so the CI action will only look at your changes.

**Comment by myoung34** [comment]
Didnt realize that was fixed upstream.
Rebased

**Pull request merged**